autohooks/plugins/pylint/__version__.py

```diff
@@ -0,0 +1,5 @@
+# pylint: disable=invalid-name
+
+# THIS IS AN AUTOGENERATED FILE. DO NOT TOUCH!
+
+__version__ = "20.4.0.dev1"
```

Comment:
Should be September now
```suggestion
__version__ = "20.9.0.dev1"
```